src/main/java/frc/robot/subsystems/swerve/SwerveDrive.java

```diff
@@ -1,75 +1,152 @@
-  private final SwerveModule backRightModule;
-
-//  private final SwerveDriveKinematics m_kinematics = new SwerveDriveKinematics(DriveConstants.frontRight, DriveConstants.frontLeft, DriveConstants.backRight, DriveConstants.backLeft);
-
-  protected Gyro gyro;
-  public Odometry odometry;
-
-  public SwerveDrive() {
-    setName("SwerveDrive");
-    gyro = new Gyro();
-
-    // initialize swerve mods (possibly move into a list for conciseness eventually)
-    frontLeftModule = new SwerveModule(PortConstants.kFrontLeftDriveMotorPort, PortConstants.kFrontLeftSteerMotorPort, false, false, PortConstants.kFrontLeftCANCoderPort, 0, false);
-    frontRightModule = new SwerveModule(PortConstants.kFrontRightDriveMotorPort, PortConstants.kFrontRightSteerMotorPort, false, false, PortConstants.kFrontRightCANCoderPort, 0, false);
-    backLeftModule = new SwerveModule(PortConstants.kBackLeftDriveMotorPort, PortConstants.kBackLeftSteerMotorPort, false, false, PortConstants.kBackLeftCANCoderPort, 0, false);
-    backRightModule = new SwerveModule(PortConstants.kBackRightDriveMotorPort, PortConstants.kBackRightSteerMotorPort, false, false, PortConstants.kBackRightCANCoderPort, 0, false);
-
-    // initialize classes which require Swerve
-    odometry = new Odometry(this);
-  }
-  public void drive(double xSpeed, double ySpeed, double rot, boolean fieldRelative) {
-
-    SwerveModuleState[] swerveModuleStates = DriveConstants.kDriveKinematics.toSwerveModuleStates(
-      fieldRelative
-        ? ChassisSpeeds.fromFieldRelativeSpeeds(xSpeed, ySpeed, rot, gyro.getYaw())
-        : new ChassisSpeeds(xSpeed, ySpeed, rot));
-
-    SwerveDriveKinematics.desaturateWheelSpeeds(swerveModuleStates, AutoConstants.kMaxSpeedMetersPerSecond);
-    frontRightModule.setDesiredState(swerveModuleStates[0]);
-    frontLeftModule.setDesiredState(swerveModuleStates[1]);
-    backLeftModule.setDesiredState(swerveModuleStates[2]);
-    backRightModule.setDesiredState(swerveModuleStates[3]);
-  }
-
-  public void zeroDrive() {
-    frontLeftModule.zeroDriveMotor();
-    frontRightModule.zeroDriveMotor();
-    backLeftModule.zeroDriveMotor();
-    backRightModule.zeroDriveMotor();
-  }
-
-  public Rotation2d getHeading() {
-    return odometry.getHeading();
-  }
-
-  public Pose2d getPoseMeters() {
-    return odometry.getPoseMeters();
-  }
-
-  public SwerveModulePosition[] getPositions() {
-    SwerveModulePosition[] positions = new SwerveModulePosition[4];
-    positions[0] = frontLeftModule.getPosition();
-    positions[1] = frontRightModule.getPosition();
-    positions[2] = backLeftModule.getPosition();
-    positions[3] = backRightModule.getPosition();
-    return positions;
-  }
+    private final SwerveModule frontLeftModule;
+    private final SwerveModule frontRightModule;
+    private final SwerveModule backLeftModule;
+    private final SwerveModule backRightModule;
+
+
+
+    private Odometry odometry;
+    protected Gyro gyro;
+
+    public SwerveDrive() {
```

Comment:
Fix the formatting on this file as well so that the diff isn't so large. There's no reason to change the indentation.